pkg/instance/service.go

```diff
@@ -184,8 +188,55 @@ func (s service) unlink(id uint) error {
 	return s.instanceRepository.Unlink(instance)
 }
 
+func (s service) matchRequiredParameters(stackParams []model.StackRequiredParameter, instanceParams []model.InstanceRequiredParameter) error {
+	stackRequiredParameterNames := make([]string, len(stackParams))
```

Comment:
Names in Go are usually chosen to be shorter than what I am used to from other languages. See https://github.com/golang/go/wiki/CodeReviewComments#variable-names (https://go.dev/doc/effective_go#names)

Maybe

```suggestion
	paramNames := make([]string, len(stackParams))
```

would do as well 😄